File: app/src/main/java/com/gadgetsfolk/kidsgk/adapter/QuestionFragmentPagerAdapter.java
package com.gadgetsfolk.kidsgk.adapter;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentStatePagerAdapter;

import com.gadgetsfolk.kidsgk.activity.QuizActivity;
import com.gadgetsfolk.kidsgk.fragment.QuizFragment;

public class QuestionFragmentPagerAdapter extends FragmentStatePagerAdapter {

    public QuestionFragmentPagerAdapter(FragmentManager fm){
        super(fm);
    }

    @NonNull
    @Override
    public Fragment getItem(int position){
        Bundle bundle = new Bundle();
        bundle.putInt("position", position);
        QuizFragment quizFragment = new QuizFragment();
        quizFragment.setArguments(bundle);
        quizFragment.setArguments(bundle);
        return quizFragment;
    }

    @Override
    public int getCount() {
        return QuizActivity.mQuizList.size();
    }
}

File: app/src/main/java/com/gadgetsfolk/kidsgk/adapter/QuestionFragmentPagerAdapter.kt
package com.gadgetsfolk.kidsgk.adapter

import android.os.Bundle
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentStatePagerAdapter
import com.gadgetsfolk.kidsgk.activity.QuizActivity
import com.gadgetsfolk.kidsgk.fragment.QuizFragment

class QuestionFragmentPagerAdapter(fm: FragmentManager?) : FragmentStatePagerAdapter(fm!!) {
    override fun getItem(position: Int): Fragment {
        val bundle = Bundle()
        bundle.putInt("position", position)
        val quizFragment = QuizFragment()
        quizFragment.arguments = bundle
        quizFragment.arguments = bundle
        return quizFragment
    }

    override fun getCount(): Int {
        return QuizActivity.mQuizList!!.size
    }
}